torch/csrc/jit/passes/alias_analysis.cpp

```diff
@@ -682,6 +686,19 @@ void AliasDb::analyzeSetAttr(Node* node) {
   registerWrite(self, node);
 }
 
+// Custom ops may write to any input and produce wildcards
+void AliasDb::analyzeCustomOp(Node* node) {
+  for (const auto input : node->inputs()) {
+    registerWrite(input, node);
```

Comment:
Could you add a check in `registerWrite` that the value needs annotation ? Otherwise we'll be disabling constant propagation on primitive values used in custom ops.